tutorials/best_practices/godot_notifications.rst

```diff
@@ -160,6 +181,27 @@ delta time methods as needed.
 
     }
 
+  .. code-tab:: cpp C++
+
+    using namespace godot;
+
+    class MyNode : public Node {
+    public:
+        // Called every frame, even when the engine detects no input.
+        void _process(double p_delta) {
+            if (Input::get_singleton->is_action_just_pressed("ui_select"))
+                UtilityFunctions::print(p_delta);
+        }
+
+        // Called during every input event. Equally true for _input().
+        void _unhandled_input(const Ref<InputEvent> &event) {
+            const InputEventKey *key_event = Object::cast_to<const InputEventKey>(*event);
+            if (key_event != nullptr && Input::get_singleton->is_action_just_pressed("ui_accept")) {
```

Comment:
```suggestion
            if (key_event.is_valid() && Input::get_singleton->is_action_just_pressed("ui_accept")) {
```

Reply:
If this is the correct way to do this Example::_input in example.cpp in the godot-cpp repository should probably also be changed, that is where I got the cast_to thing from to begin with.